decentraland/unity-explorer
# fix: scene account switch

## What does this PR change?

Fixes #1187 

The fix consists on destroy all the current scenes and also restart the realm, so the next time the world is loaded, all the scenes can be created from scratch.
Also, a refactor has been made over the realm classes to reduce coupling and cyclic dependencies limitations.

## How to test the changes?

1. Follow issue steps
2. After re-authenticating the scene should load normally
3. Check that the /reload command still works

## Our Code Review Standards

https://github.com/decentraland/unity-renderer/blob/master/docs/code-review-standards.md
